frontend/public/components/utils/_list-input.scss

```diff
@@ -1,5 +1,6 @@
 .co-list-input__add-btn {
   padding-left: 0;
+  margin-top: 10px;
```

Comment:
This adds too much whitespace when there's no help text. It would be better to adjust the margin on the help-text itself.